firmware/config/stm32f7ems/efifeatures.h

```diff
@@ -6,262 +6,75 @@
  * @date Aug 29, 2013
  * @author Andrey Belomutskiy, (c) 2012-2017
  */
+ 
+#include "../stm32f4ems/efifeatures.h"
```

Comment:
STM32F7 config is now inherited from STM32F4. This is more consistent way to support these config 'branches' and add new features.